lib/evaluate/character.h

```diff
@@ -70,6 +70,15 @@ template<int KIND> class CharacterUtils {
     return str;
   }
 
+  static Character Pad(const Character &str, std::size_t length) {
+    auto oldLength{str.length()};
+    if (length > oldLength) {
+      return str + Character(length - oldLength, Space());
+    } else {
+      return str;
```

Comment:
What if `length < oldLength`?

Reply:
Right, `Pad` interface was bad. I have renamed the function to `Resize` and it now trims the rightmost characters in such case.